fl4health/checkpointing/client_module.py

```diff
@@ -1,31 +1,37 @@
 from enum import Enum
 from logging import INFO
-from typing import Dict, Optional, Sequence, Union
+from typing import Any, Dict, Sequence, Union
 
 import torch.nn as nn
 from flwr.common.logger import log
 from flwr.common.typing import Scalar
 
-from fl4health.checkpointing.checkpointer import TorchCheckpointer
+from fl4health.checkpointing.checkpointer import PerRoundStateCheckpointer, TorchModuleCheckpointer
 
-CheckpointModuleInput = Optional[Union[TorchCheckpointer, Sequence[TorchCheckpointer]]]
+CheckpointModuleInput = Union[TorchModuleCheckpointer, Sequence[TorchModuleCheckpointer]] | None
```

Comment:
Is this type needed? Might it be clearer to just have the more explicit typing. If not maybe we can change this type name to something that communicates that there can be multiple checkpointers